src/index.js

```diff
@@ -55,8 +54,7 @@ function makeLoader(callback) {
 async function loader(source, inputSourceMap, overrides) {
   const filename = this.resourcePath;
 
-  let loaderOptions = loaderUtils.getOptions(this);
-
+  let loaderOptions = this.getOptions();
```

Comment:
https://webpack.js.org/api/loaders/#thisgetoptionsschema `loadContext#getOptions` is available only in webpack 5+, we have to drop webpack 4 support, too.